docs/data-model.rst

```diff
@@ -123,7 +126,23 @@ Each link reference is represented by an object that specifies:
     + sources: Sources
    }
 
+   object(Instructions) {
+    + programCounter: String
+    + opcode: Integer
+    + pushData: String
+    + sourceRanges: String
+    + meta: Object
+   }
+
+  object(Meta) {
+    + cost: Integer
+    + pops: Integer
+    + pushes: Integer
+    + dynamic: Integer
+   }   
 
+   Bytecode *-- "1" Instructions
```

Comment:
So I did the "collection resource" thing for `Sources`, for instance, but I'm backpedaling on that now. I think we should get rid of that and instead only have direct `"n"` relations. So in this situation, I'd expect this to be either:

```
Bytecode *-- "1" Instructions
Instructions *-- "n" Instruction
```

or what I now prefer:
```
Bytecode *-- "n" Instruction
```

What do you think?

Reply:
I think that makes sense. Changed it to the second way:
 
`Bytecode *-- "n" Instruction`